mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
since it is so maya specific could it be in mindbender.lib or should it stay in the userSetup.py